OpenEdition Journals.js

```diff
@@ -28,96 +48,92 @@ function detectWeb(doc, url) {
 		}
 	}
 
-	if (ZU.xpath(doc, '//div[@id="inside"]/div[@class="sommaire"]\
-			/dl[@class="documents"]/dd[@class="titre"]/a').length ||
-		ZU.xpath(doc, '//ul[@class="summary"]//div[@class="title"]/a').length) {
+	if (ZU.xpath(doc, '//div[@id="inside"]/div[@class="sommaire"]/dl[@class="documents"]/dd[@class="titre"]/a').length
+		|| ZU.xpath(doc, '//ul[@class="summary"]//div[@class="title"]/a').length) {
 		return "multiple";
-	} else if (ZU.xpath(doc, '//h1[@id="docTitle"]/span[@class="text"]').length ||
-		url.match(/document\d+/)) {
+	}
+	else if (ZU.xpath(doc, '//h1[@id="docTitle"]/span[@class="text"]').length
+		|| url.match(/document\d+/)) {
```

Comment:
avoid using `.match()` for testing regex. use either `/regex/.test(string)` or `string.search(regex) != -1`

Reply:
it's fixed